scripts/test-check-false-positives.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+cleanup() {
+  r=$?
+
+  # Dump stack logs
+  elastic-package stack dump -v --output "build/elastic-stack-dump/check-${PACKAGE_UNDER_TEST:-${PACKAGE_TEST_TYPE:-httpjson_false_positive_asserts}}"
+
+  # Take down the stack
+  elastic-package stack down -v
+
+  # Clean used resources
+  for d in test/packages/${PACKAGE_TEST_TYPE:-false_positives}/${PACKAGE_UNDER_TEST:-httpjson_false_positive_asserts}/; do
+    (
+      cd $d
+      elastic-package clean -v
+    )
+  done
+
+  # This is a false positive scenario and tests that the test case failure is a success scenario  
+  if [ "${PACKAGE_TEST_TYPE:-false_positives}" == "false_positives" ] && [ "${PACKAGE_UNDER_TEST:-httpjson_false_positive_asserts}" == "httpjson_false_positive_asserts" ]; then
+    if [ $r == 1 ]; then
+        exit 0
+      elif [ $r == 0 ]; then
+        exit 1
+      fi
+  fi
+
+  exit $r
+}
+
+trap cleanup EXIT
+
+export ELASTIC_PACKAGE_LINKS_FILE_PATH="$(pwd)/scripts/links_table.yml"
+
+OLDPWD=$PWD
+# Build/check packages
+for d in test/packages/${PACKAGE_TEST_TYPE:-false_positives}/${PACKAGE_UNDER_TEST:-httpjson_false_positive_asserts}/; do
+  (
+    cd $d
+    elastic-package check -v
+  )
+done
+cd -
+
+# Update the stack
+elastic-package stack update -v
+
+# Boot up the stack
+elastic-package stack up -d -v
+
+elastic-package stack status
+
+# Run package tests
+eval "$(elastic-package stack shellinit)"
+
+for d in test/packages/${PACKAGE_TEST_TYPE:-false_positives}/${PACKAGE_UNDER_TEST:-httpjson_false_positive_asserts}/; do
```

Comment:
In `pipeline.trigger.integration.tests.sh` script, PACKAGE_UNDER_TEST is not being set for the loop where this package has been added.

If it is needed, it must be added the `-p` parameter here in this line

https://github.com/elastic/elastic-package/pull/1333/files#diff-4e2e09cf3f17805ad30f828b15ea267ba7a186ff3f5129b9ecba76ef7317718bR41

Currently, when PACKAGE_UNDER_TEST is being set is because it is required/needed to just execute one package per stage in Jenkins, as it is done for the test packages under `test/packages/parallel`. These packages have special requirements (e.g. logs are updated to a special artifact buckets).

If it is needed to execute every package under `false_positives` in a different stage, it could be added a new loop there in that script for this false_positives packages. Maybe something like:

```bash
pushd test/packages/false_positives > /dev/null
for package in $(find . -maxdepth 1 -mindepth 1 -type d) ; do
    package_name=$(basename ${package})
    echo "      - label: \":go: Running integration test: ${package_name}\""
    echo "        key: \"integration-false_positives-${package_name}\""
    echo "        command: ./.buildkite/scripts/integration_tests.sh -t test-check-packages-false-positives -p ${package_name}"
    echo "        agents:"
    echo "          provider: \"gcp\""
    echo "        artifact_paths:"
    echo "          - build/test-results/*.xml"
    echo "          - build/elastic-stack-dump/check-*/logs/*.log"
    echo "          - build/elastic-stack-dump/check-*/logs/fleet-server-internal/**/*"
done
popd > /dev/null
```

Reply:
These `false_positive` tests should just run as `System tests` for any other package, just that the outcome is inversed.

I have add the `PACKAGE_UNDER_TEST` in https://github.com/elastic/elastic-package/pull/1333/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R81 

and it seems to run as a separate suite atleast. Not sure if it would be necessary to run each package separately? Do you think the the testrunner exits on execution failure / the script failure? Like `exit 1 from test execution`  or `exit 1 from script execution` 


Or Did I misunderstand ?

<img width="1228" alt="Screenshot 2023-07-03 at 11 48 14" src="https://github.com/elastic/elastic-package/assets/123897612/7847ca89-d57d-47bb-be84-995f242b2a08">

Reply:
I think I got your point now.. 

But as it is now , I don't see a need to execute each package in a separate buildKite stage. As they are just another system tests.. And I think the value of `PACKAGE_UNDER_TEST` is correctly taken in each execution as per https://buildkite.com/elastic/elastic-package/builds/1099#01890bc4-ce48-4b8b-9638-8ae44f58e102/131-2282

Reply:
Added another CI step